### PR TITLE
Toggle Show/Hiding of PR Status

### PR DESCRIPTION
**Pull request by jcox250**
Updated 'Check My Status' button so that it shows and hides your PR status.

### CODE REVIEW

File: public/js/profile.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ $(function() {
   loadIssues();
   $('#check').click(checkPRs);
   $('#share_info').change(saveShareInfo);
+  toggleStatus();
```
Comment (jcbwlkr) [inline public/js/profile.js]: Do we need this initial call to `toggleStatus`?

**Comment by jcox250** [comment]
@jcbwlkr I'd just put it in it's own function because I thought it would keep `checkPRs` cleaner but I'm happy to move it into `checkPRs` if you want?

File: public/js/profile.js (hunk 1 above)
Comment (jcox250) [reply public/js/profile.js]: No we don't actually need this now. I made an the following update to profile.css on my local version which means we shouldn't need an initial call to `toggleStatus`

```
#results {
  display: none;
}

**Comment by jcbwlkr** [comment]
@jcox250 If you don't mind I think that would be best. If fetching the PR list fails for some reason then we would want to handle that behavior differently.

**Comment by jcox250** [comment]
@jcbwlkr I just made PR #48 with these updates if you want to check them out.

**Comment by jcbwlkr** [comment]
Closing this one to focus on the other PR

**Pull request closed without merging**